tests/conftest.py

```diff
@@ -8737,6 +8843,10 @@ def finalizer():
 
 @pytest.fixture()
 def scale_noobaa_db_pod_pv_size(request):
+    return scale_noobaa_db_pv(request)
+
+
+def scale_noobaa_db_pv(request):
```

Comment:
This requires adjusting https://github.com/red-hat-storage/ocs-ci/blob/master/tests/conftest.py/#L8738-L8759 for 4.19